### DISCUSSION TITLE
I cant register deleted organization name

### DISCUSSION THREAD

**Discussion by pixelodin** [community/community#43435] [Other Feature Feedback, Questions, & Ideas]
Hi
i have deleted my organization by mistake, my main goal was to delete all repositories from my github org, when i searched online Someone on reddit said delete the organization and register it again which i did and now i cant register the same organization name again it says its unavailable.
Can someone please help me i have opened github support ticket but its been 3 days no response.

**Comment by 2lu3** [comment]
Same for me

**Comment by jose19961975** [comment]
I no

Get Outlook for Android<https://aka.ms/AAb9ysg>

________________________________

**Comment by jfullstackdev** [comment]
I don't know if that will be release after 90 days

but you can contact Support, though there is still no assurance

https://support.github.com/contact/account

**Reply by pixelodin** [reply]
i did no one is responding, Moved on now i cant wait 90 days its kinda stupid why 90 days doesnt make sense.

**Comment by ghost** [comment]
I'm in same situation. Did the name got released after 90 days?

EDIT: Here it says that account names get released after 90 days https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/deleting-your-personal-account
